api/src/routes/donate.ts

```diff
@@ -103,6 +103,15 @@ export const donateRoutes: FastifyPluginCallbackTypebox = (
         });
 
         const { email, name } = user;
+        const threeChallengesCompleted = user.completedChallenges.length >= 3;
```

Comment:
Just a thought...

Do we want to require a certification project? Three challenges isn't very many, especially with the granular step-based projects.

Reply:
Hi